src/python/aim/_sdk/remote_storage.py

```diff
@@ -531,4 +531,34 @@ def remove(self, file):
         return self._rpc_client.run_instruction(self._hash, self._handler, 'remove', (file,), is_write_only=True)
 
     def move(self, src, dest):
-        return self._rpc_client.run_instruction(self._hash, self._handler, 'move', (src, dest), is_write_only=True)
\ No newline at end of file
+        return self._rpc_client.run_instruction(self._hash, self._handler, 'move', (src, dest), is_write_only=True)
+
+
+class RemoteRepoProxy:
+    class AutoClean(RemoteResourceAutoClean):
+        PRIORITY = 60
+
+    def __init__(self, client: 'Client'):
+        self._rpc_client = client
+
+        self.init_args = pack_args(encode_tree({}))
+        self.resource_type = 'Repo'
+
+        handler = self._rpc_client.get_resource_handler(self, self.resource_type, args=self.init_args)
+
+        self._resources = RemoteRepoProxy.AutoClean(self)
+        self._resources.rpc_client = client
+        self._resources.handler = handler
+        self._handler = handler
+
+    def _delete_run(self, hash_):
```

Comment:
what about removing the `_`?

Reply:
which `_`?

Reply:
the one in the method name. just make those methods a part of public API

Reply:
there's already a public method named `delete_run`, this is one is here for the tricks with remote and local